pyvo/registry/tests/test_regtap.py

```diff
@@ -779,12 +795,14 @@ def getflashcol(name):
 
 
 @pytest.mark.remote_data
-def test_sia_registry_searches():
-    # SIA2 services, e.g. Spitzer SEIP were originally not found by the registry search
-    image_services_v1 = regsearch(servicetype='sia')
-    image_services_v2 = regsearch(servicetype='sia2')
-
-    assert image_services_v1 != image_services_v2
-
-    assert len([s.ivoid for s in image_services_v2 if 'spitzer/images/seip' in s.ivoid]) > 0
-    assert len([s.ivoid for s in image_services_v1 if 'spitzer/images/seip' in s.ivoid]) == 0
```

Comment:
why remove these tests?

Reply:
Well, they're transferring quite a bit of data with IMHO relatively little increase in possible mishaps covered, and I'm not particularly fond of the assertion that v1 and v2 services must be different.  Sure, it will always be true, but that's sociology, not computer science.

If what you want to test is that sia and sia2 yield different queries, let's test for that in a local test (actually, there's tests in test_rtcons.py that taken together ensure that).

If what you want to test is that dual-interface services like seip are mapped in a sensible way, I'd say let's additionally constrain by identifier='ivo://irsa.ipac/spitzer/images/seip' and make sure we have a result in each case.

Me, I think both kinds of logic are covered by other tests, but I can easily be swayed.  What I'd much rather avoid unless there's a really compelling need: pulling potentially multi-megabyte query results into unit tests.